ipol_demo/modules/dispatcher/test.py

```diff
@@ -47,22 +47,13 @@ class DispatcherTests(unittest.TestCase):
     #####################
     #       Tests       #
     #####################
-    def test_ping(self):
-        """
-        Test ping
-        """
-        status = None
-        try:
-            ping = '{}/api/dispatcher/ping'.format(self.BASE_URL)
-            response = requests.get(ping).json()
-            status = response.get('status')
-        finally:
-            self.assertEqual(status, 'OK')
-
     def test_find_suitable_demorunner(self):
         """
         Test get suitable demorunner
         """
+        if True:
+            # test broken since the endpoint get_suitable_demorunner was removed
```

Comment:
What is preventing to remove the test now?

Reply:
Tests have been reimplemented in commit cbd91cf99f38a5edd0553a83859df3c23a0cf667.
Sorry for the large PR, I think it's good to review it both by commit and globally.

Reply:
I can rewrite the history to remove this "if True" in order to keep the commits clean.

Reply:
I saw later about the other commits, looks good to me 👍🏼